package-templates/react/package.json

```diff
@@ -24,7 +24,8 @@
     "build": "rollup --config rollup.config.js",
     "start": "../../scripts/start-storybook",
     "start:ssr-preview": "../../scripts/start-storybook --ssr-preview",
-    "test": "../../scripts/run-tests"
+    "test": "../../scripts/run-tests",
+    "updatePackageLock": "npm i --package-lock-only"
```

Comment:
`build-package-lock` ?

Reply:
i think we separate words with `:` in npm scripts, so something like `update:lock` would do